handlers/mostPopular.js

```diff
@@ -0,0 +1,27 @@
+const {
+	checkIfFileExists,
+	readFile
+} = require('../utils/supportFunctions')
+
+/**
+ * Function is responsible for returning the most popular joke in the joke.json file
+ * @returns string containing joke
+ */
+exports.mostPopularhandler = async () => {
+	try {
+		// Check if file exists
+		const fileExists = await checkIfFileExists()
+
+		// If there is no file throw error
+		if (!fileExists)
+			throw new Error('The file dosnt exist')
+
+		// read file
+		const parsedData = await readFile()
+
+		// return top joke
+		return JSON.stringify(parsedData['top'])
+	} catch (err) {
+		throw err
```

Comment:
same as above we don't handle errors at the lower level. So if the file couldn't be read for some reason, I think it would be good to handle it in try catch